independent-publisher-connectors/Udemy/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Client ID",
+          "description": "The Client ID for this api",
+          "tooltip": "Provide the Client ID",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Client Secret",
+          "description": "The Client Secret for this api",
+          "tooltip": "Provide the Client Secret",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#a435f0",
+    "capabilities": [],
+    "publisher": "Nanddeep Nachan, Smita Nachan"
```

Comment:
@nanddeepn Please add the stackOwner property.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties